docs/core/propagation.md

```diff
@@ -1,127 +1,126 @@
 # Propagation and extrapolation
 
-:::{attention}
-This section is largely **outdated** and will be replaced in the future.
+The track propagation is an essential part of track reconstruction. This section describes the high-level classes and concepts used for this task in Acts.
+
+## Overview: Steppers, Navigators and Actors
+
+The propagation through a geometry is based on the interaction of two different components:
+
+* The **Stepper** provides the implementation of the numerical integration of the equation of motion in the magnetic field.
```

Comment:
it's not always numerical integration, probably one can say `the solution of the equation of motion (either by analytical means or through numerical integration)`